### ISSUE TITLE
Implement list component

### ISSUE THREAD

**Issue by jeluard** [status-im/status-components#4]
Implement `List` component as defined by https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=787%3A1108

It's already been implement in ReactNative/ClojureScript [here](https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/list/views.cljs). Port it to JavaScript.

Also see https://github.com/status-im/status-react/issues/7785

Depends on #3

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 80.0 DAI (80.0 USD @ $1.0/DAI)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/status-im/status-components/4/2682).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $52,526.06 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [anshumanv](https://gitcoin.co/profile/anshumanv) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=anshumanv) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=anshumanv))_.**

Experienced React dev, will port the component to React, have worked with status in the past.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by grigio** [comment]
Hi, do it depends on other components? like togglebutton or others?

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 12 months from now.
Please review their action plans below:


**1) [elemino](https://gitcoin.co/profile/elemino) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=elemino) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=elemino))_.**

Seem pretty straight forward. I would like to take a stab at this. Thank you.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by gitcoinbot** [comment]
@elemino Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=1) | [3 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=3) | [5 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=5) | [10 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=10) | [100 days](https://gitcoin.co/issue/status-im/status-components/4/2682?snooze=100)

**Comment by pbteja1998** [comment]
@jeluard is this available?

**Comment by jeluard** [comment]
@pbteja1998 It it! :)

**Comment by pbteja1998** [comment]
@jeluard I am unable to apply through `gitcoin`

**Comment by Elemino** [comment]
Hey guys, Can you provide a little bit of background on what instance of the app I should run so I can start applying the changes. Also, where in Figma should I look specifically in order to implement the design? Lastly, I would like to know if there's a preferred way that this project uses to implement CSS in js. Thanks.

**Comment by Elemino** [comment]
Hello, do you still see the value in turning around this bounty in storybook? I have time to put effort into this and hopefully, I can finish in a few days. I've made some progress with clojurescript and there's also new progress being done here: https://github.com/status-im/status-react/issues/7785. Please do let me know so I can prioritize because it's been a while. Thanks a lot!

**Comment by jeluard** [comment]
@Elemino it's not a priority anymore, sorry :(

**Comment by shyamks** [comment]
Is the issue active ? What is the status ?

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 5 months, 3 weeks from now.
Please review their action plans below:


**1) [ashokk1990](https://gitcoin.co/profile/ashokk1990) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=ashokk1990) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=ashokk1990))_.**

First I will setup Environment on my local and then I will test Already implement React Native/Clojure version. if it has no issues then I will start porting it into javascript. I will keep funder's posted every alternate day.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 4 months, 1 week from now.
Please review their action plans below:


**1) [vashist99](https://gitcoin.co/vashist99) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=vashist99) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=vashist99))_.**

This is an issue that seems doable within my skill set. I will Start off by creating a React.js component for  each of the list items. Since the dimensions are specified clearly, it would be very comfortably done.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 3 months, 2 weeks from now.
Please review their action plans below:


**1) [vashist99](https://gitcoin.co/vashist99) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=vashist99) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=vashist99))_.**

This is an issue that seems doable within my skill set. I will Start off by creating a React.js component for  each of the list items. Since the dimensions are specified clearly, it would be very comfortably done.
**2) [hrithikgautham](https://gitcoin.co/hrithikgautham) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=hrithikgautham) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=hrithikgautham))_.**

Ive had a good grasp of react JS. Porting from React Native to JS will a doable task.

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__Workers have applied to start work__.


These users each claimed they can complete the work by 2 months, 2 weeks ago.
Please review their action plans below:


**1) [vashist99](https://gitcoin.co/vashist99) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=vashist99) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=vashist99))_.**

This is an issue that seems doable within my skill set. I will Start off by creating a React.js component for  each of the list items. Since the dimensions are specified clearly, it would be very comfortably done.
**2) [ir3ne](https://gitcoin.co/ir3ne) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=ir3ne) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=ir3ne))_.**

is it valid again? the issue is closed on github.
Thanks
**3) [licarijd](https://gitcoin.co/licarijd) has applied to start work _(Funders only: [approve worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=approve&worker=licarijd) | [reject worker](https://gitcoin.co/issue/status-im/status-components/4/2682?mutate_worker_action=reject&worker=licarijd))_.**

I'll port the component. Is this still available?

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/status-im/status-components/4/2682).

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 80.0 DAI  (80.0 USD @ $1.0/DAI) attached to this issue has been **cancelled** by the bounty submitter__

 
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $135,670.37 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
